### PR TITLE
update bluecryptor to 1.0.2. add swift 4.1 support

### PR DESCRIPTION
**Pull request by quanvo87**
## Checklist:

- [ ] I have submitted a [CLA form](https://github.com/IBM-Swift/CLA)
- [ ] If applicable, I have updated the documentation accordingly.
- [ ] If applicable, I have added tests to cover my changes.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ let package = Package(
         // .package(url: /* package url */, from: "1.0.0"),
         .package(url: "https://github.com/IBM-Swift/BlueSocket.git", .upToNextMinor(from: "0.12.0")),
         .package(url: "https://github.com/IBM-Swift/BlueSSLService.git", .upToNextMinor(from: "0.12.0")),
-        .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .upToNextMinor(from: "0.8.0")),
+        .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .upToNextMinor(from: "1.0.0")),
```
Comment (ianpartridge) [inline Package.swift]: You might like to consider loosening this to just `.from("1.0.0")`

Comment (quanvo87) [reply Package.swift]: How about `upToNextMajor`?

Comment (ianpartridge) [reply Package.swift]: Same thing, I think.

Comment (quanvo87) [reply Package.swift]: Oh okay. I'll do that. `from` just sounds a bit scary in that it would pick up a new major as well.

**Pull request merged**